app/bundles/IntegrationsBundle/Sync/DAO/Mapping/UpdatedObjectMappingDAO.php

```diff
@@ -26,16 +26,23 @@ class UpdatedObjectMappingDAO
      */
     private $objectModifiedDate;
 
+    /**
+     * @var mixed
```

Comment:
Isn't internal object ID always int?
```suggestion
     * @var int
```